src/Adapter/SqlManager/SqlQueryValidator.php

```diff
@@ -84,6 +84,10 @@ private function getErrors(array $sqlErrors)
         $errors = [];
 
         foreach ($sqlErrors as $key => $sqlError) {
+            if (false === is_array($sqlError)) {
+                $sqlError = [];
```

Comment:
Should it be `$sqlError = [$sqlError];`?

Reply:
I'm not 100% sure, but the usecase we handle here is `$sqlError` is a boolean and your idea would not go well with what happens next in the code
https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/SqlManager/SqlQueryValidator.php#L82

Reply:
Ok I understand what you mean =)